Vale fixes (#199)
botpress/docs: learn/guides/advanced/additional-information/limits-and-quotas-of-botpress-cloud.mdx

# Limits and Quotas of Botpress Cloud

> At Botpress, there're a set of limits and quotas to ensure the stability of our platform and the bots created by our customers and the community.

Author: this isn't vale's suggested fix, but the suggested fix was lousy.

> At Botpress, there're a set of limits and quotas to ensure the stability of our platform and the bots created by our customers and the community.

Reviewer: ```suggestion
Botpress sets certain limits and quotas to ensure the stability of the platform and the bots created by customers and the community.
```
